src/js/providers/html5.js

```diff
@@ -173,6 +173,12 @@ define([
         _videotag = _videotag || document.createElement('video');
         _videotag.className = 'jw-video jw-reset';
 
+        // prevent browser from showing second cast icon, using presentation-api
```

Comment:
Thanks for fixing this so fast!

"disableRemotePlayback" attribute is a part of Remote Playback API, not Presentation API: https://w3c.github.io/remote-playback/

Happy to chat more about these two APIs and how they relate to each other :)